tests/test_message.py

```diff
@@ -267,6 +267,60 @@ class Color(proto.Enum):
     expected_dict = {"mass_kg": 20}
     assert s_dict_2 == expected_dict
 
+    s_dict_2 = Squid.to_dict(s_new_2, always_print_fields_with_no_presence=False)
+    expected_dict = {"mass_kg": 20}
+    assert s_dict_2 == expected_dict
+
+    s_dict_2 = Squid.to_dict(
+        s_new_2,
+        including_default_value_fields=False,
+        always_print_fields_with_no_presence=False,
+    )
+    expected_dict = {"mass_kg": 20}
+    assert s_dict_2 == expected_dict
+
+    s_dict_2 = Squid.to_dict(
+        s_new_2,
+        including_default_value_fields=True,
+    )
+    expected_dict = {"mass_kg": 20, "chromatophores": []}
+    assert s_dict_2 == expected_dict
+
+    s_dict_2 = Squid.to_dict(
+        s_new_2,
+        always_print_fields_with_no_presence=True,
+    )
+    expected_dict = {"mass_kg": 20, "chromatophores": []}
+    assert s_dict_2 == expected_dict
+
+    s_dict_2 = Squid.to_dict(
```

Comment:
Can we add a test case here where we don't pass either parameter?

Reply:
Done in https://github.com/googleapis/proto-plus-python/pull/433/commits/9d85d8bf84b861333385ac30d7512a6d0d211309